Automattic/woocommerce-payments
# Update Jetpack packages to latest versions

Fixes #3348
Closes #1243, #3090

### NOTE: THIS PR SHOULD BE ONLY MERGED AFTER THE TEST PACKAGE `3.3.0-test-3` IS CREATED.

#### Changes proposed in this Pull Request

Update three current Jetpack package:

- https://github.com/Automattic/jetpack-connection/blob/master/CHANGELOG.md
- https://github.com/Automattic/jetpack-config/blob/master/CHANGELOG.md
- https://github.com/Automattic/jetpack-autoloader/blob/master/CHANGELOG.md

I had a look at them and they're mostly related to devs and other dependencies. In other words, they're generally safe under my eyes.









#### Testing instructions





* Check the differences between new and old versions in the changelog links above.
* Try onboarding a new merchant with the live server API (don't use the local server as it disables Jetpack authentication), check out a few orders, check transactions and deposits.
* Ideally, check browser console and PHP error logs, and ensure there is no relevant error.

-------------------

- [x] ~Covered with tests (or have a good reason not to test in description ☝️)~ Manual tests by reviewers and GlobalStep. Critical flows should be well-cover this PR.

**Post merge**



- [x] Added testing instructions to the [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) (or does not apply) => No need as critical flows should be enough.
